libs/SmartStore/Configuration/SmartStore-Common.xcconfig

```diff
@@ -11,8 +11,8 @@ GCC_PREFIX_HEADER = $(PROJECT_NAME)/$(PROJECT_NAME)-Prefix.pch
 
 PRODUCT_NAME = SmartStore
 
-OTHER_CFLAGS = -DSQLITE_HAS_CODEC -DFMDatabase=SF_FMDatabase -DFMStatement=SF_FMStatement -DFMDatabasePool=SF_FMDatabasePool -DFMDatabaseQueue=SF_FMDatabaseQueue -DFMResultSet=SF_FMResultSet -DFMDBBlockSQLiteCallBackFunction=SF_FMDBBlockSQLiteCallBackFunction $(inherited)
-OTHER_LDFLAGS = -ObjC -lsqlcipher
```

Comment:
I thought it did with dynamic frameworks, but I could be wrong. I'll add it back in for now, until I validate.